infra/bicep/app.bicep

```diff
@@ -28,7 +28,7 @@ var llmFastModelFullName = toLower('${llmFastModel}-${llmFastVersion}')
 var llmSlowModelFullName = toLower('${llmSlowModel}-${llmSlowVersion}')
 var embeddingModelFullName = toLower('${embeddingModel}-${embeddingVersion}')
 var cosmosContainerName = 'calls-v3'  // Third schema version
-var localConfig = loadYamlContent('../config.yaml')
+var localConfig = loadYamlContent('../configs/config.yaml')
```

Comment:
I understand you are trying to put all the configuration in a single folder. But, there is only one config file required to run the app, `config.yaml`. What the point to put a single file in a new folder?